server/scripts/randomize-participants.ts

```diff
@@ -1,18 +1,46 @@
 import { faker } from '@faker-js/faker';
 import _ from 'lodash';
 import { convertToCsv } from './services/participant-seed';
-import { Program } from '../constants';
+import { healthRegions, Program, yesOrNo } from '../constants';
 
 // this is for canadian postal codes
 faker.locale = 'en_CA';
 
-const DEFAULT_PREF_LOCATION = 'Fraser';
```

Comment:
I remember we used `Fraser` for a reason, don't remember if it was just for simplicity's sake though

Reply:
You're right. It caused a lot of problems, but I just fixed the participants.